tests/system_tests_edge_router.py

```diff
@@ -1887,6 +1887,8 @@ def test_50_link_topology(self):
         bc_a.close()
         bc_b.close()
 
+        self.INT_B.wait_address_unsubscribed("CfgLinkRoute1")
```

Comment:
It would be better to do this check in test_51_link_route_proxy_configured to keep it all in one place and prevent accidental breakage if - for whatever reason - tests need to be re-ordered.  Eventually all tests that enable a link route(s) should perform that type of check at the start of the test.

Reply:
Update: Jiri's approach is much cleaner - require all tests to verify that any addresses it introduced are removed from the router at the end of the individual test.  But it would need to be done to all applicable tests, which means... more work.

Reply:
For now, I am going to commit this fix to see what is happening on the main branch. If we see more of these kinds of failures, let's then think of an approach to solving the issue in a more generic fashion